storybook/index.css

```diff
@@ -58,10 +100,6 @@
   line-height: 1.5;
   font-weight: 400;
 
-  color-scheme: light dark;
-  color: rgba(255, 255, 255, 0.87);
-  background-color: #242424;
-
```

Comment:
> [!note]
> I removed this because the component wasn't visible in the preview. It might need to be restored based on project needs, it just wasn't clear to me why we have this.